H/HiGHS/bundled/patches/v1.3.0.patch

```diff
@@ -0,0 +1,68 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index e5d3a14b8..2112a7e54 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -125,9 +125,16 @@ enable_cxx_compiler_flag_if_supported("-Wno-format-truncation")
+ enable_cxx_compiler_flag_if_supported("-pedantic")
+ endif()
+ 
+-check_cxx_compiler_flag("-mpopcnt" FLAG_MPOPCNT_SUPPORTED)
+-set(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -mpopcnt")
+-
++if(CMAKE_SYSTEM_PROCESSOR STREQUAL "aarch64")
++  message("FLAG_MPOPCNT_SUPPORTED is not available on aarch64")
++  set(FLAG_MPOPCNT_SUPPORTED OFF)
```

Comment:
It may actually be better to match x86_64/i686 and powerpc, and do nothing for all the rest, whatever it is (instead of matching all other possible architectures, which are unbound)

Reply:
Also, it'd be great to upstream this patch :slightly_smiling_face:

Reply:
Yeah once we get something working, I'll send it to @jajhall and he can release v1.3.1